File: Java/opentraceability/src/main/java/opentraceability/models/events/EventLocation.java
package opentraceability.models.events;

import opentraceability.models.identifiers.GLN;
import opentraceability.utility.attributes.OpenTraceabilityAttribute;

public class EventLocation {

    @OpenTraceabilityAttribute(ns = "", name = "id")
    public GLN gln;

    public EventLocation() {}

    public EventLocation(GLN gln) {
        this.gln = gln;
    }
}

File: Java/OpenTraceability/src/main/opentraceability/models/events/eventlocation.kt
package opentraceability.models.events

import opentraceability.models.identifiers.GLN
import opentraceability.utility.attributes.OpenTraceabilityAttribute

class EventLocation {

    @OpenTraceabilityAttribute("","id")
    var GLN: GLN? = null

    constructor(gln: GLN) {
        this.GLN = gln;
    }

}
